frontend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:14-alpine as builder
```

Comment:
current-alpine didn't work. Not sure why so I used 14.

Reply:
Komisch, dass es nicht ging, aber sollte auch mit Node 14 passen.